ansible/roles/atmo-user-ssh-keys/tasks/main.yml

```diff
@@ -1,11 +1,26 @@
 ---
 
 - name: Add users ssh keys to authorized_keys
-  authorized_key: user=root key="{{ item }}" state=present
+  authorized_key:
+    user: root
+    state: present
+    key: "{{ item }}"
   with_items: USERSSHKEYS
   become: yes
   when: USERSSHKEYS is defined
 
+# See https://github.com/ansible/ansible/issues/11257 , https://github.com/ansible/ansible-modules-core/issues/1515
+- name: verify that home directory of user exists (to avoid ansible error)
+  stat:
+    path: "/home/{{ ATMOUSERNAME }}"
+  register: user_home_dir
+  when: ATMOUSERNAME is defined
```

Comment:
The user _should_ have a home directory by the time this playbook executes, in a normal execution flow.

 I encountered this in a very specific corner case and the solution above got me through.